android/build.gradle

```diff
@@ -11,8 +11,8 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-  compileSdkVersion 23
-  buildToolsVersion "23.0.1"
+  compileSdkVersion 25
+  buildToolsVersion "25.0.1"
```

Comment:
@lucasbento It would require people to download SDK 25, which is not as lightweight as only updating build tools (several Go)